### DISCUSSION TITLE
Having problem with obfuscated stacktrace

### DISCUSSION THREAD

**Discussion by subhajitxyz** [oppia/oppia-android#5476] [Q&A (Other)] [answered]
I am facing an issue with the obfuscation of stack traces while trying to solve a bug. Can we deobfuscate the stack trace, or is there any other solution

**Comment by adhiamboperes** [comment]
@subhajitxyz, could you please clarify your issue? And give us an example screenshot or stacktrace if possible

**Reply by subhajitxyz** [reply]
Actually i am working on this issue #5449

**Reply by adhiamboperes** [reply, marked as answer]
Since this.is a production stacktrace, there isn't much we can do to unobfuscate it. What you can do is to make a best effort attempt to reproduce the bug using the little information that's actually available. Should this be difficult, I would suggest working on a different issue.
